lib/ReactCrop.js

```diff
@@ -521,14 +524,14 @@ class ReactCrop extends Component {
 				<div className='ReactCrop--drag-bar ord-s' data-ord='s'></div>
 				<div className='ReactCrop--drag-bar ord-w' data-ord='w'></div>
 
-				<div className='ReactCrop--drag-handle ord-nw' data-ord='nw'></div>
-				<div className='ReactCrop--drag-handle ord-n' data-ord='n'></div>
-				<div className='ReactCrop--drag-handle ord-ne' data-ord='ne'></div>
-				<div className='ReactCrop--drag-handle ord-e' data-ord='e'></div>
-				<div className='ReactCrop--drag-handle ord-se' data-ord='se'></div>
-				<div className='ReactCrop--drag-handle ord-s' data-ord='s'></div>
-				<div className='ReactCrop--drag-handle ord-sw' data-ord='sw'></div>
-				<div className='ReactCrop--drag-handle ord-w' data-ord='w'></div>
+				{ellipse ? null : <div className='ReactCrop--drag-handle ord-nw' data-ord='nw' />}
```

Comment:
Just a style thing but can we invert these and just use `&&`?
e.g. {!ellipse && <div ...}`